placeholder-sql-azure-pipelines.yml

```diff
@@ -55,7 +53,7 @@ stages:
   - template: azure-pipelines-templates/build/job/dacpac-template.yml@das-platform-building-blocks
     parameters:
       SolutionBaseName: $(DACPACSolutionBaseName)
-      SqlProjectPath: src/$(DACPACSolutionBaseName)/$(DACPACSolutionBaseName).sqlproj
+      SqlProjectPath: src/$(SolutionBaseName).Database/$(SolutionBaseName).Database.sqlproj
```

Comment:
will need to change the line above as $(DACPACSolutionBaseName) no longer exists